shared/fs/common/progress.js

```diff
@@ -30,19 +32,19 @@ const stylesTubeBox = {
   marginRight: globalMargins.xtiny,
 }
 
-const stylesTube = {
+const stylesTube = memoize(width => ({
   backgroundColor: globalColors.black_20,
   borderRadius: 4.5,
   height: 4,
   marginTop: 3,
-  width: 40,
-}
+  width,
+}))
 
-const stylesTubeStuffing = {
-  ...stylesTube,
+const stylesTubeStuffing = width => ({
```

Comment:
Since the stuffing one has width calculated from `completePortion` which can be different every time and likely won't benefit much from memoizing, but still wastes memory for storing the memoized objects.

Reply:
You're still passing in `width` from `ProgressProps` into `stylesTubeStuffing`, then changing the `width` after that.

Reply:
Ah you're right! Thanks. will fix